compiler/luci-interpreter/src/kernels/DepthwiseConv2D.cpp

```diff
@@ -58,6 +59,17 @@ void DepthwiseConv2D::configure()
   {
     LUCI_INTERPRETER_CHECK(bias() == nullptr || bias()->element_type() == DataType::S32);
   }
+  else if (input()->element_type() == DataType::S8 && filter()->element_type() == DataType::S8)
+  {
+    LUCI_INTERPRETER_CHECK(filter()->shape().num_dims() == 4);
+    LUCI_INTERPRETER_CHECK(filter()->shape().dim(3) == filter()->scales().size());
+    for (auto zerop : filter()->zero_points())
+    {
+      LUCI_INTERPRETER_CHECK(zerop == 0);
+    }
+    LUCI_INTERPRETER_CHECK(filter()->shape().dim(0) == 1);
```

Comment:
There is also the same check below in the code:
`LUCI_INTERPRETER_CHECK(filter_shape.dim(0) == 1);`

Reply:
```suggestion
```